lms/static/js/fixtures/discovery.html

```diff
@@ -2,26 +2,28 @@
   <div id="discovery-form" class="wrapper-search-context">
     <div id="discovery-message" class="search-status-label"></div>
     <form class="wrapper-search-input">
-      <input class="discovery-input" placeholder="Search for a course" type="text"/><!-- removes spacing
-      --><button type="submit" class="button postfix discovery-submit" aria-label="Search">
+      <input class="discovery-input" placeholder="Search for a course" type="text"/>
+      <button type="submit" class="button postfix discovery-submit" aria-label="Search">
         <i class="icon fa fa-search" aria-hidden="true"></i>
         <div aria-live="polite" aria-relevant="all">
           <div id="loading-indicator" class="loading-spinner hidden">
-          <i class="icon fa fa-spinner fa-spin" aria-hidden="true"></i>
+          <i class="icon fa fa-spinner fa-spin"></i>
           <span class="sr">Loading</span>
           </div>
         </div>
       </button>
     </form>
-    
+
     <div id="filter-bar" class="filters hide-phone">
     </div>
 
-    <div class="courses" role="region" aria-label="${_('List of Courses')}">
+    <div class="courses" role="region" aria-label="List of Courses">
       <ul class="courses-listing"></ul>
     </div>
 
     <aside aria-label="Refine your search" class="search-facets phone-menu">
+      <h2 class="header-search-facets">Refine your search</h2>
```

Comment:
Refine Your Search (changing to sentence caps to be consistent with other UI text)